src/main/java/org/sagebionetworks/bridge/config/Config.java

```diff
@@ -0,0 +1,30 @@
+package org.sagebionetworks.bridge.config;
+
+import java.util.List;
+
+public interface Config {
```

Comment:
Is it necessary to have an interface here? The value of the config package is the shared implementation and file format, not the generic interface.